launch/tier4_planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_out/pull_out.param.yaml

```diff
@@ -5,12 +5,12 @@
       th_stopped_velocity: 0.01
       th_stopped_time: 1.0
       collision_check_margin: 1.0
-      pull_out_finish_judge_buffer: 1.0
+      collision_check_distance_from_end: 15.0
+      pull_out_finish_judge_buffer: 0.0
```

Comment:
Is this parameter required?
If it is not necessary, it may be good to remove this parameter.

Reply:
yes, removed it in https://github.com/autowarefoundation/autoware.universe/pull/2335/commits/9989939c47d0b08ef9131cf55a756d60d115ce33